### ISSUE TITLE
R&D Console lagging when using search function searching

### ISSUE THREAD

**Issue by techno0** [tgstation/tgstation#61551]
## Round ID:
 169822

## Testmerges:

#61509: '[TM Candidate] Overhauls orbit and POI code to fix part of issue #61508 where players could observe /mob/living/new_player on the lobby.' by Timberpoes at commit 2c4ba30d32
#61419: 'Radically improves explosion logging.' by Timberpoes at commit 79cc0be467
## Reproduction:

- Open R&D Console
- Use Search
- Type one letter
- 4 Second game freeze
- Type one letter 
- 4 Second game freeze

this goes on forever

**Comment by Mothblocks** [comment]
not again

**Comment by Sealed101** [comment]
yes again and its worse than before

**Comment by Mothblocks** [comment]
makes sense i suppose, popper mounts are still slow, just their updating is waht i fixed

i think you could fix *most* of the lag if you made the components that represent an individual rnd node purecomponents
